test/components/fetchers/test_link_content_fetcher.py

```diff
@@ -46,10 +46,11 @@ def test_init(self):
         assert fetcher.retry_attempts == 2
         assert fetcher.timeout == 3
         assert fetcher.handlers == {
-            "text/html": _text_content_handler,
-            "text/plain": _text_content_handler,
-            "application/pdf": _binary_content_handler,
-            "application/octet-stream": _binary_content_handler,
+            "text/*": _text_content_handler,
+            "application/json": _text_content_handler,
+            "application/*": _binary_content_handler,
+            "audio/*": _binary_content_handler,
+            "video/*": _binary_content_handler,
```

Comment:
Maybe its helpful to add a test for an edge case as well, like:
`assert fetcher._resolve_handler("unknown/type") == _text_content_handler
`

Reply:
Check out this one as well - the unknown type is usually designated by [application/octet-stream](https://www.perplexity.ai/search/what-is-a-nyy_z.wMQgCp6affVmUNsA#0) and we have application/* handler registered already to be a _binary_content_handler 👍

Reply:
Right! That makes sense. Thanks for sharing the informative link.